runtime/slice.go

```diff
@@ -156,6 +156,58 @@ func initSliceType(map[string]*Object) {
 	SliceType.slots.Repr = &unaryOpSlot{sliceRepr}
 }
 
+func sliceIndex(f *Frame, o *Object) (i int64, raised *BaseException) {
```

Comment:
That's odd. maxIntBig is [defined as grumpy.MaxInt](https://github.com/google/grumpy/blob/master/runtime/numeric.go#L35). sys.maxsize is [defined as sys.maxint](https://github.com/google/grumpy/blob/master/lib/sys.py#L28), which is `__go__.grumpy.MaxInt`. So maxIntBig should be the same as sys.maxsize.